<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QClient</class>
 <widget class="QWidget" name="QClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QClient</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>400</width>
      <height>700</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active/>
      <inactive/>
      <disabled/>
     </palette>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab{
    margin:0px;
    width:48px;
	height:32px;
	border: none;

	border-top: 1px solid #bebebe;
	font : 12pt;color:white;
	 
}
QTabBar::tab:hover{
	background-color: rgb(209, 196, 233);
	border-color:white;
}</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset>
       <normaloff>C:/Users/hasee/Downloads/聊天 (2).png</normaloff>C:/Users/hasee/Downloads/聊天 (2).png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QListWidget" name="m_pMessageListWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>700</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset>
       <normaloff>C:/Users/hasee/Downloads/用户.png</normaloff>C:/Users/hasee/Downloads/用户.png</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
   </widget>
   <widget class="QListWidget" name="listWidget_ChatList">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>780</width>
      <height>480</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_MessageInput">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>541</y>
      <width>781</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SendMessage">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1084</x>
      <y>652</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>103</red>
          <green>58</green>
          <blue>183</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>103</red>
          <green>58</green>
          <blue>183</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>103</red>
          <green>58</green>
          <blue>183</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>103</red>
          <green>58</green>
          <blue>183</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>103</red>
          <green>58</green>
          <blue>183</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>103</red>
          <green>58</green>
          <blue>183</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>103</red>
          <green>58</green>
          <blue>183</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>103</red>
          <green>58</green>
          <blue>183</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>103</red>
          <green>58</green>
          <blue>183</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_SendMessage
{
	background-color: rgb(103, 58, 183);
	border-radius : 2px;
}
#pushButton_SendMessage:hover
{
	
	
	background-color: rgb(81, 45, 168);
}
</string>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
